packages/xrpl/src/models/methods/nftsByIssuer.ts

```diff
@@ -0,0 +1,68 @@
+import { NFToken } from '../common'
+
+import { BaseRequest, BaseResponse, LookupByLedgerRequest } from './baseMethod'
+
+/**
+ * The nfts_by_issuer method returns a list of NFTokens issued by the account.
+ * The order of the NFTs is not associated with the date the NFTs were minted.
+ * Expects a response in the form of a {@link
+ * NFTsByIssuerResponse}.
+ *
+ * @category Requests
+ */
+export interface NFTsByIssuerRequest
+  extends BaseRequest,
+    LookupByLedgerRequest {
+  method: 'nfts_by_issuer'
+  /**
+   * A unique identifier for the account, most commonly the account's address
+   */
+  issuer: string
+  /**
+   * Value from a previous paginated response. Resume retrieving data where
+   * that response left off. This value is stable even if there is a change in
+   * the server's range of available ledgers.
+   */
+  marker?: unknown
+  /**
+   * Filter NFTs issued by this issuer that have this taxon.
+   */
+  nft_taxon?: number
+  /**
+   * Default varies. Limit the number of transactions to retrieve. The server
+   * is not required to honor this value.
+   */
+  limit?: number
+}
+
+/**
+ * Expected response from an {@link NFTsByIssuerRequest}.
+ *
+ * @category Responses
+ */
+export interface NFTsByIssuerResponse extends BaseResponse {
```

Comment:
same with the response, it should be exposed to methods and clients

Reply:
I wonder whether it should be exposed to client/index.ts or not. I checked rest 2 clio related file (nftinfo and nfthistory), both of them don't have anything written in client/index.ts (but appear in methods/index.ts)